lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -31,20 +31,19 @@ class InfoCommand < HammerCLIForeman::InfoCommand
 
       PROVIDER_SPECIFIC_FIELDS = {
         'ovirt' => [
-          Fields::Field.new(:label => _('UUID'), :path => ["compute_resource", "uuid"])
+          Fields::Field.new(:label => _('UUID'), :path => [:uuid])
         ],
         'ec2' => [
-          Fields::Field.new(:label => _('Region'), :path => ["compute_resource", "region"])
+          Fields::Field.new(:label => _('Region'), :path => [:region])
         ],
         'vmware' => [
-          Fields::Field.new(:label => _('UUID'), :path => ["compute_resource", "uuid"]),
-          Fields::Field.new(:label => _('Server'), :path => ["compute_resource", "server"])
+          Fields::Field.new(:label => _('UUID'), :path => [:datacenter])
```

Comment:
1. Api doesn't return attribute called uuid here is sample api response(few field truncated)
```
{"description":"","url":"vmware.example.redhat.com","name":"vmware","provider":"Vmware","provider_friendly_name":"VMware","user":"admin","datacenter":"Datacenter","server":"vmware.example.com","set_console_password":true,"images":[],"compute_attributes":[],"locations":[{"id":2,"name":"location","title":"location","description":null}],"organizations":[{"id":3,"name":"org","title":"org","description":""}]}
```
Then what should be the value of uuid?

2. Running compute resource create without uuid says "Datacenter can't be blank"
```
hammer  compute-resource create --provider="VMWare" --name "vCenter2" --description "Test vCenter"   --user "user" --password "password"  --url "vmware.example.com"
Could not create the compute resource:
  Datacenter can't be blank
```

But after passing uuid='datacenter' allows to create vmware compute resource and works.

```
hammer compute-resource create --provider="VMWare" --name "vCenter2" --description "Test vCenter"   --user "user" --password "password"  --url "vmware.example.com" --uuid Datacenter1
Compute resource created
```
so its looks like api accept uuid as datacenter.